backend/routes/forms/submit.py

```diff
@@ -168,16 +168,45 @@ async def submit(self, request: Request) -> JSONResponse:
             if any("unittests" in question.data for question in form.questions):
                 unittest_results = await execute_unittest(response_obj, form)
 
-                if not all(test.passed for test in unittest_results):
-                    # Return 500 if we encountered an internal error (code 99).
-                    status_code = 500 if any(
-                        test.return_code == 99 for test in unittest_results
-                    ) else 403
+                failures = []
+                status_code = 403
```

Comment:
More appropriate [HTTP code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422).
```suggestion
                status_code = 422
```